### PR TITLE
Lower owner hashing activation slot for devnet

### LINKED ISSUE #10163: verify_bank_hash failed: MismatchedAccountHash: Tip of master is still not compatible with devnet/mainnet-beta
I believe the last remaining rolling update to v1.1 has now activated and master/v1.1 should be compatible again.  However starting a master validator, thusly:
```
solana-validator \
    --trusted-validator 5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on \
    --no-untrusted-rpc \
    --ledger ledger-testnet \
    --rpc-port 8899 \
    --dynamic-port-range 8000-8010 \
    --entrypoint 35.203.170.30:8001 \
    --expected-genesis-hash 4uhcVJyU9pJkvQyS88uRDiswHXSCkY3zQawwpjk2NsNY \
    --expected-shred-version 56096 \
    --enable-rpc-transaction-history \
    -o -
```
produces a bank hash mismatch on snapshot loading, eg:
```
WARN  solana_runtime::accounts] verify_bank_hash failed: MismatchedAccountHash
thread 'main' panicked at 'Snapshot bank for slot 14786165 failed to verify', ledger/src/snapshot_utils.rs:478:9
```

*Status update is here: https://github.com/solana-labs/solana/issues/10163#issuecomment-632604747*

### PR DESCRIPTION
**Pull request by ryoqun**
#### Problem

devnet hasn't enabled owner-included hashing due to its relatively low slot (5_850_000).

testnet (tds) has fully transitioned to the new hashing.

#### Summary of Changes

Lower the activation slot to gradually transition to the new hashing on devnet. This should take 2-4 days on devnet.

Contexts:

- https://github.com/solana-labs/solana/pull/10166/files#r428900953
- https://github.com/solana-labs/solana/issues/10163#issuecomment-632604747

Fixes #10163 

After devnet fully transitioned, we can address this: #10227.

### CODE REVIEW

**Comment by ryoqun** [comment]
@mvines I'm sure you're very busy for the upgrade party. :) But could you apply this for devnet within the timeframe of 6_500_000? If not, I'm happy to adjust the slot.

This change shouln't affect testnet(tds) at all. And this isn't urgent at all as long as the compatibility isn't problematic between v1.2(devnet) and v1.3(master) branch.

File: runtime/src/accounts_db.rs [hunk 1]
```diff
@@ -1209,8 +1209,10 @@ impl AccountsDB {
     }
 
     pub fn include_owner_in_hash(slot: Slot) -> bool {
-        // Account hashing updated to include owner activates at this slot on the testnet
-        slot >= 14_000_000
+        // Account hashing will be updated to include owner at this slot on the devnet.
+        // For testnet, it fully transitioned already thanks to eager rent collection,
+        // so, this check is irrelevant, strictly speaking.
+        slot >= 6_500_000
```
Comment (mvines) [inline runtime/src/accounts_db.rs:1215]: I've rolled devnet back a bit as a part of upgrading it to 1.2, so we can go with a much earlier slot:
```suggestion
        slot >= 5_800_000
```

**Comment by solana-grimes** [comment]
:broken_heart: Unable to automerge due to CI failure

**Comment by mvines** [comment]
devnet is now running 1.2.0, which contains this PR

**Pull request merged**